Source/Core/VideoCommon/FrameDumpFFMpeg.cpp

```diff
@@ -68,7 +68,7 @@ AVRational GetTimeBaseForCurrentRefreshRate()
   int num;
   int den;
   av_reduce(&num, &den, int(vi.GetTargetRefreshRateDenominator()),
-            int(vi.GetTargetRefreshRateNumerator()), std::numeric_limits<int>::max());
+            int(vi.GetTargetRefreshRateNumerator()), std::numeric_limits<unsigned short>::max());
```

Comment:
std::numeric_limits<int>::max() is generally correct here (num/den are ints in AVRational), except when using the default lossy codec (i.e. MPEG-4 codec), which specifically (unlike every single other codec in ffmpeg) does not like that, and instead has a max of 2^16-1 (i.e. std::numeric_limits<uint16_t>::max()). This extra reduction should only happen in the case of that codec being used (although even then you're getting an imprecise rational, so for long videos there will eventually be some drift, best that's kept only for the MPEG-4 codec).